### DISCUSSION TITLE
How to adjust shutter speed using picamera2 library

### DISCUSSION THREAD

**Discussion by CHEONGYW235** [raspberrypi/picamera2#760] [General]
Hi, I just got a raspberry pi global shutter camera, and I wonder how to I adjust the shutter speed using the picamera2 library? Thanks.

**Comment by davidplowman** [comment]
Use
```
picam2.set_controls({'ExposureTime': 10000})
```
where the value is given in microseconds. It takes a few frames for the value to take effect if you do this while the camera is already running. If you do it just before starting the camera (`picam2.start()`), then it will take effect immediately.
